gtsam/slam/ProjectionFactor.h

```diff
@@ -99,6 +99,17 @@ namespace gtsam {
     /** Virtual destructor */
     ~GenericProjectionFactor() override {}
 
+    /// @return the measured pixel coordinates
+    auto measured() const -> const Point2 & { return measured_; }
```

Comment:
`measured()` is already defined below (line 173). You might want to move that up here instead?